edx_exams/apps/core/api.py

```diff
@@ -142,6 +143,23 @@ def update_attempt_status(attempt_id, to_status):
     return attempt_id
 
 
+def delete_exam_attempt(attempt, requesting_user):
+    """
+    Delete or 'reset' an exam attempt
```

Comment:
We often refer to a delete as 'reset' so that's why I included it. That said, I think you have a point here. This doesn't just delete and it also triggers other things on platform. It would be better to just rename this to reset_exam_attempt

Reply:
Thanks for clarifying! And sounds good.